packages/asset-swapper/CHANGELOG.json

```
@@ -1,4 +1,17 @@
 [
+    {
+        "version": "3.1.0",
```

```suggestion
        "version": "4.0.0",
```

due to removal of public function `getSmartContractParamsOrThrow`.